dashboard/modules/state/state_head.py

```diff
@@ -455,6 +456,19 @@ async def summarize_objects(self, req: aiohttp.web.Request) -> aiohttp.web.Respo
         record_extra_usage_tag(TagKey.CORE_STATE_API_SUMMARIZE_OBJECTS, "1")
         return await self._handle_summary_api(self._state_api.summarize_objects, req)
 
+    @routes.get("/api/v0/tasks/timeline")
```

Comment:
Btw there was a proposal to use api/v0/tasks?format=perfetto. I decided to use a separate endpoint due to 2 reasons

1. It has a completely different schema from regular list tasks
2. It should support the downloading header (to support downloading from the browser), which is also not a common pattern from list tasks.

So I thought having a separate endpoint makes more sense here.